torchvision/prototype/transforms/_deprecated.py

```diff
@@ -18,6 +18,7 @@ class ToTensor(Transform):
     _transformed_types = (PIL.Image.Image, np.ndarray)
 
     def __init__(self) -> None:
+        # FIXME: should the replacement be a `Compose` with `ConvertImageDtype`?
```

Comment:
Let's remove this fixme. Previously we have intentionally pushed people away from ToTensor which does 2 things at the same time (rescaling to 0-1 and tensor conversion). We asked people to explicitly use 2 transforms instead, so by design we didn't want to offer a single transform that does both. The reason is, we don't have to scale the image to 0-1 prematurely.

The new API allows people to keep the images on the original 0-255 scale even more, so adding this convenience method doesn't align with this. Let's remove the FIXME.

Reply:
After an offline discussion with @pmeier, he clarified that the addition of the Copose would be just on the example inside the warning. I thought the proposal was for a new shortcut in our code. We should do what Philip suggested here to ensure we inform the users about the best practice.